README.md

```diff
@@ -133,6 +133,21 @@ command_output = test_vmi.virt_launcher_pod.execute(command="command-example")
 ```
 If no Pod was found, will raise ``ResourceNotFoundError``.
 
+### NNCP Capture Syntax
+Using capture syntax to switch ipv4 config between interfaces
+```python
+with NodeNetworkConfigurationPolicy(
+    name = "capture_nncp",
```

Comment:
please remove spaces before and after `=`

Reply:
ACK